WeakAurasOptions/AuthorOptions.lua

```diff
@@ -2275,23 +2284,27 @@ local function mergeOptions(mergedOptions, data, options, config, prepath, paren
     -- find the best place to start inserting the next option to merge
     local nextToMerge = options[i]
     local shouldMerge = false
-    for j = nextInsert, #mergedOptions + 1 do
-      local mergedOption = mergedOptions[j]
-      if not mergedOption then
-        break
-      end -- no more options to check, so must insert
-      local validMerge = true
-      for field in pairs(significantFieldsForMerge) do
-        if nextToMerge[field] ~= mergedOption[field] then
-          validMerge = false
+    if not nextToMerge.noMerge then
+      for j = nextInsert, #mergedOptions + 1 do
+        local mergedOption = mergedOptions[j]
+        if not mergedOption then
+          break
+        end -- no more options to check, so must insert
+        local validMerge = mergedOption.noMerge
```

Comment:
shouldn't this be "         local validMerge = *not* mergedOption.noMerge "

Reply:
You're right, fixed.